src/providers/app-config/app-config.model.ts

```diff
@@ -17,6 +17,7 @@ export type AppConfig = {
   },
   approvedDeviceIdentifiers: string[],
   timeTravelDate?: string,
+  role: string;
```

Comment:
Same as other could we have the roles in an enum DE and LDTM

Reply:
I'm not sure that it is worth it, as the value being provided will be controlled by what's in Dynamo. Also, we are not limited to DE and LDTM. This may grow in the future.

Reply:
For further context, we aren't storing a boolean in Dynamo, it's a string representation of the role.

Reply:
It just makes the code a bit safer and is easier to reference/use it throughout the app. I can see the person who picks up the ticket and has to limit components by roles preferring an enum over strings.

Cause it may change in the future is not really a valid reason as everything can change in the future so we'd do no work ;)

Reply:
Assuming that there may be other roles in the system (in the future), having an enum already in place would be helpful. Ans we don't need to worry about case-sensitivity when checking the role